compiler-plugin/src/test/scala/scalapb/compiler/FieldTransformationsSpec.scala

```diff
@@ -512,6 +533,26 @@ class FieldTransformationsSpec extends AnyFlatSpec with Matchers with ProtocInvo
       msg = "int32: {gt: 0, lt: 0}",
       pattern = "int32: {gt: 0}"
     ) must be(true)
+
```

Comment:
Can you add to docs/transformations.md a sentence describibing the semantics of matching repeated fields? I.e. empty vs non-empty is compared for presence. 
The behavior for CONTAINS can be a bit surprising, because it compares the lists for equality, not for containment. The behavior make sense, but let's document it.

The code looks good.